src/utils/perfagg.py

```diff
@@ -86,6 +87,121 @@
 }
 
 
+# joins disparate runs less dumbly than rocprof
+def join_prof(workload_dir, out):
+    files = glob.glob(workload_dir + "/" + "pmc_perf_*.csv")
+    df = None
+
+    for i, file in enumerate(files):
+        # _df = parse_rocprof_kernels(file)
+        _df = pd.read_csv(file)
+        key = _df.groupby("KernelName").cumcount()
+        _df["key"] = _df.KernelName + " - " + key.astype(str)
```

Comment:
We probably want two versions of this key calculation here:

1. What we have here, essentially "unique kernel name dispatches"
2. Adding another " - " + 'grd' to get the total grid size, that gives us ~ what we discussed last time (join by kernel grid size + name + dispatch id)

Ideally, we can switch between the two (not sure what the default should be, perhaps the 'grid' strategy?) via a command line arg